src/common/transformations/src/transformations/common_optimizations/optimize_strided_slice.cpp

```diff
@@ -366,25 +368,72 @@ bool ov::pass::GroupedSliceToVSplitOptimization::run_on_model(const std::shared_
     return graph_rewritten;
 }
 
+ov::pass::SliceSequenceToSingleSlice::SliceSequenceToSingleSlice() {
+    MATCHER_SCOPE(SliceSequenceToSingleSlice);
+    using namespace ov::op;
+    using namespace ov::op::util;
+    using namespace ov::pass::pattern;
+
+    auto const_axes_1_pattern = wrap_type<v0::Constant>();
+    auto const_axes_2_pattern = wrap_type<v0::Constant>();
+    auto slice_1_pattern =
+        wrap_type<v8::Slice>({any_input(), any_input(), any_input(), any_input(), const_axes_1_pattern},
+                             consumers_count(1));
+    auto slice_2_pattern =
+        wrap_type<v8::Slice>({slice_1_pattern, any_input(), any_input(), any_input(), const_axes_2_pattern});
+
+    ov::matcher_pass_callback callback = [=](Matcher& m) {
+        const auto& pattern_to_output = m.get_pattern_map();
+        auto slice_1 = pattern_to_output.at(slice_1_pattern);
```

Comment:
You're using get_pattern_map() here. I saw, other transformations make use of get_pattern_value_map(). As I understand, there's no big difference as the first maps to nodes, the latter to output. Or are there any other reasons to use one over another?

Reply:
discussed offline